desktop/plugins/network/ManageMockResponsePanel.tsx

```diff
@@ -27,6 +27,11 @@ import {RequestId} from './types';
 import {message} from 'antd';
 import {NUX, Layout} from 'flipper-plugin';
 
+import {Modal} from 'antd';
+import {ExclamationCircleOutlined} from '@ant-design/icons';
```

Comment:
I think this is unused?

Reply:
Addressed it in phabricator

Reply:
I removed the code (and also combined the imports).  Hopefully, this is not in conflict with phabricator (what is phabricator?).

Reply:
At least at some point, it was an internal Facebook tool. I think it allowed to review code, track bugs, etc...

Reply:
yeah it is kinda our internal version of Github